Bottom: 6fe12a9388185384d9e1b8e7615a887326e434bd
Top:    ec614a3502d50deafe328ff5ed5416d092052c1f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 22:09:44 +0100

Refresh of hid-gtk-remove-unused-variable

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 074ef9b..f33678a 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -465,24 +465,18 @@ get_grid_value_index (gboolean allow_fail)
 static void
 h_adjustment_changed_cb (GtkAdjustment * adj, GhidGui * g)
 {
-  gdouble xval, yval;
-
   if (g->adjustment_changed_holdoff)
     return;
-  xval = gtk_adjustment_get_value (adj);
-  yval = gtk_adjustment_get_value (GTK_ADJUSTMENT (ghidgui->v_adjustment));
+
   ghid_port_ranges_changed ();
 }
 
 static void
 v_adjustment_changed_cb (GtkAdjustment * adj, GhidGui * g)
 {
-  gdouble xval, yval;
-
   if (g->adjustment_changed_holdoff)
     return;
-  yval = gtk_adjustment_get_value (adj);
-  xval = gtk_adjustment_get_value (GTK_ADJUSTMENT (ghidgui->h_adjustment));
+
   ghid_port_ranges_changed ();
 }
